src/test/java/uk/co/ramp/covid/simulation/output/network/ContactPairWithLocationTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.co.ramp.covid.simulation.output.network;
+
+
+import org.junit.Test;
+import uk.co.ramp.covid.simulation.place.CommunalPlace;
+import uk.co.ramp.covid.simulation.place.Shop;
+import uk.co.ramp.covid.simulation.population.Adult;
+import uk.co.ramp.covid.simulation.testutil.SimulationTest;
+
+import static org.junit.Assert.assertEquals;
+import static uk.co.ramp.covid.simulation.population.Person.Sex.FEMALE;
+import static uk.co.ramp.covid.simulation.population.Person.Sex.MALE;
+
+public class ContactPairWithLocationTest extends SimulationTest {
+
+    @Test
+    public void testCompare() {
+
+        Adult p1 = new Adult(30, FEMALE);
+        Adult p2 = new Adult(40, MALE);
+        Shop s1 = new Shop(CommunalPlace.Size.SMALL);
+        ContactPairWithLocation cp1 = new ContactPairWithLocation(p1, p2, s1);
+
+        Adult p3 = new Adult(30, FEMALE);
+        Adult p4 = new Adult(40, MALE);
+        Shop s2 = new Shop(CommunalPlace.Size.SMALL);
+        ContactPairWithLocation cp2 = new ContactPairWithLocation(p3, p4, s2);
+
+        ContactPairWithLocation cp3 = cp1;
+
+        //These contact pairs are not comparable
+        assertEquals(-1, cp1.compareTo(cp2));
+
+        //These contact pairs are comparable
+        assertEquals(0, cp3.compareTo(cp1));
```

Comment:
I'm not sure about the use of "compareTo" here (especially the comment, since cp1 and cp2 _are_ comparable, just that cp1 < cp2).

I'm also not convinced that compareTo is every actually used (@johnnonweiler ?). I tried adding a break point and it never seemed to be called (I guess the idea was to spit out the network ordered?). This might highlight some deeper issue that we should investigate.

Reply:
I think you're correct that compareTo is not being used.  I don't think indicates any deeper issues - I did some experimentation with ordered Map types, so was using it then, but then ended up using a HashMap.  (I don't mind if we leave it in, because it might be useful in future, or take it out, because it is not being used.)

Reply:
Maybe the comment is wrong. It's testing that cp1 and cp2 are not the same objects